symbols/arm9.yml

```diff
@@ -292,6 +292,16 @@ arm9:
         
         r0: number
         return: number // 256
+    - name: sin_4096
```

Comment:
This isn't `sin(x)`, it's `sin(abs(x))` so we should name it accordingly

Reply:
Renamed to `SinAbs4096`.